chalk/core.py

```diff
@@ -146,19 +146,6 @@ def _repr_svg_(self) -> str:
         return svg
 
     def atop(self, other: "Diagram") -> "Diagram":
-        """Given two diagrams ``a`` and ``b``, ``a.atop(b)``
-        places ``a`` and ``b`` such that none of them move.
-        This is equivalent to **union** operation of two the
-        shapes.
-
-        💡 ``a.atop(b)`` is equivalent to ``a + b``.
-
-        Args:
-            other (Diagram): Another diagram object.
-
-        Returns:
-            Diagram: A diagram object.
-        """
```

Comment:
Why are you removing this? Is it incorrect? Since, `Diagram` class will be used everywhere, doesn't it make sense to provide docstrings here?

Having examples somewhere else could be helpful. But that should not make docstrings redundant. Shouldn't it?

Reply:
These comments are really good, but I moved them to the version of this function in `__init__`. https://github.com/danoneata/chalk/pull/58/files#diff-ebd35dd9c16e8a4cb47f718f53b6c8e9754d5c7666711be9b750956aa5ba8b57R237

The reason is that I don't think users should use this function. Instead they should either do `a + b` or `atop(a, b)`. This isn't your issue, but I think there are currently too many ways to do some of the core commands.